### ISSUE TITLE
Operational tracking of Platform Subscriptions

### ISSUE THREAD

**Issue by jefkel** [BCDevOps/developer-experience#509]
**Problem:** Platform Services Team needs to track an inventory of all hardware and softtware asses for the Openshift 4 Platform and all its clusters. 

We need to create a spreadsheet that include the following information:
 - every server that is deployed as part of any of the OCP 4 cluster is listed and is listed only once, include Openshift nodes as well as NetApp storage devices
- for every server include:
     -  the hardware details i.e. name, model, CPU core count, RAM , purpose (Worker, Master, Infra, NetApp Storage, etc)
     -  the name of the cluster is belongs too 
     - the number of Openshift licenses applied to it
     - the number and type of Next Gen Security Tool licenses applied to it i.e. 2 Aporeto enforcer licenses, 1 Aqua enforcer license, etc

Create a separate tab to list the software licenses that are applied centrally to the entire cluster (vs per node). e.g. Vault, Sysdig, etc.

**Notes: Only Openshift 4 assets should be capture in the spreadsheet, no OCP 3.11 assets should be included. I suggest to document the Platform assets as we expect they to be as of April 1, 2021 when the OCP 3.11 Platform won't exist anymore and all its hardware/software parts will have been moved to the OCP 4 clusters. **

**Definition of Done**: the spreadsheet should be created in Google Drive at https://drive.google.com/file/d/1zh_DTBWxbZ9JFk0CKevF6BYudZwuE-NL/view?usp=sharing

**Comment by jefkel** [comment]
Current projected (based on BCGov OCP4 Cluster Worksheets - post migration expectations)
Sysdig: 104 used (100 purchased)
Aporeto: 104 used (130 purchased)
Aqua (Prod): 66 used (56 purchased)
Aqua (Non-Prod): 8 used (8 purchased)
Artifactory: 10 used (7 purchased) (no gold cluster)

**Comment by tosazuwa** [comment]
@mitovskaol  & @stregear trea to take initial pass - Platform hardware/software inventory doc

**Comment by mitovskaol** [comment]
Hosting Branch does not currently have a system in place that tracks subscriptions. 

@stregear will check with a Service Manager at OCIO Service Branch if they have a system we can use.

**Comment by mitovskaol** [comment]
@lukegonis and @stregear can you please look into start collecting the hardware and subscription into in a spreadsheet and we can transfer into a proper tracking system when we find one.

**Comment by lukegonis** [comment]
@mitovskaol yes, will do.

**Comment by lukegonis** [comment]
After Security tools are installed on OCP 4, will there be any licenses remaining to expand into ARO?

**Comment by mitovskaol** [comment]
@lukegonis Can you please provide an update for this work. Thanks!

**Comment by mitovskaol** [comment]
@Joe-Craig will present the first draft of the Platform inventory at the meeting on Jan 6, 2021

**Comment by mitovskaol** [comment]
The hardware inventory part is now complete. The tracking of software licenses will be done in a separate ticket.
